iceberg/iceberg-rest-server/src/main/java/org/apache/gravitino/iceberg/service/dispatcher/IcebergTableEventDispatcher.java

```diff
@@ -79,20 +81,28 @@ public LoadTableResponse createTable(
     NameIdentifier nameIdentifier =
         IcebergRestUtils.getGravitinoNameIdentifier(
             metalakeName, context.catalogName(), tableIdentifier);
-    eventBus.dispatchEvent(
-        new IcebergCreateTablePreEvent(context, nameIdentifier, createTableRequest));
+    Optional<BaseEvent> transformedEvent =
```

Comment:
How do you make sure that this `Optional` is never empty?

Reply:
The event framework provides the guarantee, and I add the null check for the transformed pre event, it'll be a BUG if the transformed event is empty.